ios/.symlinks/plugins/integration_test

```diff
@@ -0,0 +1 @@
+/Users/ayushbherwani/Desktop/flutter/flutter/.pub-cache/hosted/pub.dartlang.org/integration_test-1.0.2+2/
```

Comment:
Do we need these in ios symlinks? Or can we revert it?

Reply:
Yes, it shouldn't be there. I'll update gitignore and remove it from git.